### PR TITLE
Added the cluster_id tag

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [x] dependencies

## Description
Added the `cluster_id` tag to the tag_cluster resources.

### CODE REVIEW

File: cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py [hunk 1]
```diff
@@ -13,10 +15,12 @@ class TagClusterResources(TagClusterOperations):
 
     SHORT_ID = 5
     NA_VALUE = 'NA'
+    CLUSTER_ID_COST_ALLOCATION_TAG = 'cluster_id'
 
     def __init__(self, cluster_name: str = None, cluster_prefix: str = None, input_tags: dict = None,
                  region: str = 'us-east-2', dry_run: str = 'yes', cluster_only: bool = False):
-        super().__init__(cluster_name=cluster_name, cluster_prefix=cluster_prefix, input_tags=input_tags, region=region, dry_run=dry_run, cluster_only=cluster_only)
+        super().__init__(cluster_name=cluster_name, cluster_prefix=cluster_prefix, input_tags=input_tags, region=region,
+                         dry_run=dry_run, cluster_only=cluster_only)
```
Comment (ebattat) [inline cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:23]: What did u change here ?

File: cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py [hunk 2]
```diff
@@ -45,6 +49,25 @@ def __append_input_tags(self, current_tags: list = None):
...
+        cluster_id_tag = get_tag_value_from_tags(tags=tags, tag_name=self.CLUSTER_ID_COST_ALLOCATION_TAG)
+        if not cluster_id_tag:
+            cluster_name = ''
+            for tag in tags:
+                if self.cluster_prefix in tag.get('Key'):
+                    cluster_name = tag['Key']
+                    break
+            if cluster_name:
+                cluster_name = cluster_name.split('/')[-1]
+                tags.append({'Key': self.CLUSTER_ID_COST_ALLOCATION_TAG, 'Value': cluster_name})
```
Comment (ebattat) [inline cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:69]: Are you sure the cluster_name is the cluster_id ?

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:69]: Screenshot 2024-05-22 at 4 38 16 PM

File: cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:23]: indentation

File: cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py (hunk 2 above)
Comment (ebattat) [reply cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:69]: cluster_name == cluster_id ?

Comment (ebattat) [reply cloud_governance/policy/policy_operations/aws/tag_cluster/tag_cluster_resouces.py:69]: Pls add comment cluster_name == cluster_id

**Review by ebattat — APPROVED** [review]
/lgtm

**Pull request merged**